validate_states.py

```diff
@@ -195,6 +196,15 @@ def get_telem_values(tstart, msids, days=14, dt=32.8, name_map={}):
     return out
 
 
+def get_bad_mask(tlm):
+    mask = np.zeros(len(tlm), dtype='bool')
+    for interval in characteristics.bad_times:
+        bad = ((tlm['date'] >= DateTime(interval['start']).secs)
```

Comment:
OK.  Thanks.  tlm is a relatively small data set and performance isn't an issue here, so yes, I think I will leave this one, but will be smarter about this for similar searches.